packages/data/src/world/mq/deku_tree_mq.yml

```diff
@@ -2,126 +2,282 @@
+    "MQ Deku Tree Grass Slingshot Room Front 1": "can_cut_grass"
+    "MQ Deku Tree Grass Slingshot Room Front 2": "can_cut_grass"
+    "MQ Deku Tree Grass Slingshot Room Back 1": "can_cut_grass"
+    "MQ Deku Tree Grass Slingshot Room Back 2": "can_cut_grass"
+"MQ Deku Tree Basement":
+  dungeon: DT
+  exits:
+    "MQ Deku Tree Lobby": "true"
+    "MQ Deku Tree Basement Ledge": "is_adult || event(DEKU_BLOCK) || trick(OOT_DEKU_SKIP) || has_hover_boots || hookshot_anywhere || climb_anywhere"
+    "MQ Deku Tree Room Before Water Room Entry Door": "true"
+  events:
+    DEKU_MQ_WATER_PATH_TORCH1: "has_fire || can_use_sticks" #You need to go through the next room with fire sticks for this event. From xeno: "cond(setting.doorShuffle !== none, has_fire, (roomClearEvent && has_sticks) || has_fire)"
+    DEKU_MQ_BASEMENT_EYE_SWITCH: "can_use_bow || can_use_slingshot"
+  locations:
+    "MQ Deku Tree Basement Chest": "can_use_sticks || has_fire"
+    "MQ Deku Tree Grass Basement Lower 1": "can_cut_grass"
+    "MQ Deku Tree Grass Basement Lower 2": "can_cut_grass"
+    "MQ Deku Tree Grass Basement Lower 3": "can_cut_grass"
+    "MQ Deku Tree Grass Basement Lower 4": "can_cut_grass"
+"MQ Deku Tree Room Before Water Room Entry Door":
+  dungeon: DT
+  exits:
+    "MQ Deku Tree Basement": "true"
+    "MQ Deku Tree Room Before Water Room Door to Basement": "event(DEKU_MQ_BASEMENT_EYE_SWITCH)"
+"MQ Deku Tree Room Before Water Room Door to Basement":
+  dungeon: DT
+  exits:
+    "MQ Deku Tree Room Before Water Room Entry Door": "event(DEKU_MQ_ROOM_BEFORE_WATER_CLEAR)"
+    "MQ Deku Tree Room Before Water Room": "true"
+"MQ Deku Tree Room Before Water Room":
+  dungeon: DT
+  exits:
+    "MQ Deku Tree Room Before Water Room Door to Basement": "true"
+    "MQ Deku Tree Room Before Water Room Door to Water Room": "true"
+  events:
+    DEKU_MQ_WATER_PATH_TORCH1: "has_fire"
+    DEKU_MQ_ROOM_BEFORE_WATER_CLEAR: "soul_deku_scrub && soul_enemy(SOUL_ENEMY_GOHMA_LARVA) && can_damage"
+  locations:
+    "MQ Deku Tree Grass Room Before Spike 1": "can_cut_grass"
+    "MQ Deku Tree Grass Room Before Spike 2": "can_cut_grass"
+    "MQ Deku Tree Grass Room Before Spike 3": "can_cut_grass"
+    "MQ Deku Tree Grass Room Before Spike 4": "can_cut_grass"
+"MQ Deku Tree Room Before Water Room Door to Water Room":
+  dungeon: DT
+  exits:
+    "MQ Deku Tree Room Before Water Room": "true"
+    "MQ Deku Tree Water Room Door to Room Before Water Room": "event(DEKU_MQ_WATER_PATH_TORCH1)"
+"MQ Deku Tree Water Room Door to Room Before Water Room":
+  dungeon: DT
+  exits:
+    "MQ Deku Tree Room Before Water Room Door to Water Room": "true"
+    "MQ Deku Tree Water Room Front": "true"
+"MQ Deku Tree Water Room Front":
+  dungeon: DT
+  exits:
+    "MQ Deku Tree Water Room Door to Room Before Water Room": "true"
+    "MQ Deku Tree Water Room Back": "true" #You can do this as both adult and child, without a shield, by rolling under the spike.
```

Comment:
I'm fairly certain the strict requirements were a mercy. Yes, you can do it with a roll, but should it be expected when it can be finnicky? Probably not. And OOTR also has it setup that way with the roll as a trick. This also goes for the event below.

Reply:
The roll is not finnicky. I stand by this; if Outside Ganon Castle GS can still just have a sword in logic, then *this* is perfectly fine.

Reply:
Still, this was also decided by AmazingAmpharos for OOTMM, who was also the main logic writer for OOTR.

Reply:
This isn't OOTR. Saying "OOTR does it" has not been a valid argument for over a year, Nax even said that.

Reply:
That isn't even the point. the point is that both Ampharos and I agreed it should be a mercy.

Reply:
I guess we could make it a trick.

But there's a huge amount of difference in difficulties when we use mercies or not. It's inconsistent.

Reply:
Trick added.